### PR TITLE
Updates for release v1.1.7

### PR DESCRIPTION
**Pull request by knihit**
*Description of changes:* 

### Security

- Updated node and python packages to patch vulnerabilities

### Fixed

- Search failure when deployed with exclusively with Amazon OpenSearch option

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by tabdunabi — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -69,7 +85,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [1.0.9] - 2024-05-16
 
-### Update
+### Changed
```
Comment (jamesnixon-aws) [inline CHANGELOG.md:88]: nit: I think these and those below would be better as Security tags, but this is fine.

**Review by jamesnixon-aws — DISMISSED** [review]

**Review by tabdunabi — APPROVED** [review]
LGTM

**Review by mukitmomin — APPROVED** [review]
LGTM

**Pull request merged**